LenovoLegionToolkit.Lib/Settings/ApplicationSettings.cs

```diff
@@ -47,6 +47,9 @@ public class ApplicationSettingsStore
         public bool SynchronizeBrightnessToAllPowerPlans { get; set; }
         public ModifierKey SmartFnLockFlags { get; set; }
         public bool ResetBatteryOnSinceTimerOnReboot { get; set; }
+        public int UpdateCheckMiniumTimeSpanHours { get; set; }
```

Comment:
I think this is a bit much. I guess a drop down with some values like hourly, daily, weekly, monthly etc.
It would be also nice to store this as an actual `TimeSpan` instead of `int`s.